web/themes/custom/votegov/src/sass/uswds-overrides/usa-button.scss

```diff
@@ -22,6 +22,7 @@
   &.usa-focus,
   &.usa-button--hover {
     background-color: $base-primary-hover;
+    outline: .25rem solid $base-primary-hover;
```

Comment:
This will add style to all buttons to have focus outside the same as the back ground color